.pylintrc

```diff
@@ -155,6 +155,8 @@ disable=bad-indentation,
         duplicate-code,
         deprecated-method,
         superfluous-parens,
+        protected-access,
```

Comment:
these are in depot_tools pylint which we should ideally be using instead of our own

Reply:
actually I removed this one because it was on the list of exceptions they wanted to remove, but in the context of a patch I think it's ok and makes sense here